planet/clients/subscriptions.py

```diff
@@ -46,22 +46,17 @@ async def list_subscriptions(self,
             ClientError
 
         """
-        # Temporary marker for behavior of module with unpatched state.
-        if _fake_subs is None:
-            raise NotImplementedError
```

Comment:
We're rolling with the placeholder implementation to not be blocked by code coverage requirements.